pkgs/by-name/me/meson/package.nix

```diff
@@ -68,6 +68,10 @@ python3.pkgs.buildPythonApplication rec {
 
     # Nixpkgs cctools does not have bitcode support.
     ./006-disable-bitcode.patch
+
+  ] ++ lib.optionals stdenv.isFreeBSD [
```

Comment:
Can we make this unconditional somehow? otherwise it could easily stop applying on update.